cms/templates/cms/program_page.html

```diff
@@ -1,12 +1,234 @@
+            <div class="container-fluid">
+              <div class="navbar-header">
+                <a class="navbar-brand" href="javascript:void(0)"><img
+                        src="/static/images/logo-mit-micromasters-whitered@2x.png"
+                        width="430" height="80"
+                        alt="MIT Micromasters"/></a>
+                <div class="nav-utility pull-right">
+                  {% if authenticated %}
+                    <a class="btn btn-danger" href="/logout">
+                      Sign out
+                    </a>
+                  {% else %}
+                    <span>Get Started Today!</span>
+                    <a class="btn btn-danger" href="{% url 'social:begin' 'edxorg' %}">
+                      Sign in with edX.org
+                    </a>
+                  {% endif %}
+                </div>
+              </div>
+              <!-- <ul class="nav navbar-toolbar navbar-left">
+                 <li class="active"><a href="">Programs</a></li>
+                 </li>
+                 </ul> -->
+            </div>
+          </nav>
+          <div class="banner-wrapper-content">
+            <h1>MIT’s #1 ranked <span>Supply Chain Management</span> program,<br> now delivered in
+              additional new ways.
+            </h1>
+            <!--<a href="add" type="button" class="btn btn-primary btn-lg">
+               <span>LEARN MORE</span>
+               </a> -->
+          </div>
+          <!--
+             <div class="banner-bottom">
+                <div class="banner-wrapper-content">
+                   <h2>Get Started Today! <a href="add" type="button" class="btn btn-success margin-left-10">
+                      <span>Sign in with edX.org</span>
+                      </a>
+                   </h2>
+                </div>
+             </div>
+             -->
+        </div>
+        <div class="panel panel-main-content">
+          <!-- Nav tabs -->
+          <ul class="nav nav-tabs nav-tabs-line nav-justified nav-tabs-animate nav-programs" role="tablist">
+            <li role="presentation" class="active">
+              <a href="#home" aria-controls="home" role="tab" data-toggle="tab">Program Home</a>
+            </li>
+            <li role="presentation">
+              <a href="#faqs" aria-controls="faqs" role="tab" data-toggle="tab">Frequently Asked
+                Questions</a>
+            </li>
+            <li role="presentation">
+              <a href="#courses" aria-controls="courses" role="tab" data-toggle="tab">Courses</a>
+            </li>
+          </ul>
+
+          <!-- Tab panes -->
+          <div class="tab-content tab-content-programs">
+            <div role="tabpanel" class="tab-pane active fade in" id="home">
+
+
+              <div class="panel-body">
+                <div class="row">
+                  <div class="col-md-8">
+                    <h3>{{ page.title }}</h3>
+
+                    <p>{{ page.description|safe }}</p>
```

Comment:
Can you change to `|richtext`? The [documentation](http://docs.wagtail.io/en/v1.5/topics/writing_templates.html) says that this will also allow the user to show images and handles internal links correctly which the `safe` filter won't do